### ISSUE TITLE
Cannot find module 'tslib'

### ISSUE THREAD

**Issue by The-Code-Monkey** [airbnb/ts-migrate#45]
How come i keep getting this error when running via npx?

**Comment by CasbaL** [comment]
So am  i
```bash
$ npx ts-migrate -- --help
npx: 449 安装成功，用时 10.22 秒
Cannot find module 'tslib'
Require stack:
- /Users/casbal/.npm/_npx/79956/lib/node_modules/ts-migrate/build/cli.js
```

> os: macOS Mojave 10.14.6
> node -v: v12.18.2
> npx -v: 6.14.5

**Comment by brandongregoryscott** [comment]
Getting this as well - npm version 6.4.1, node v8.16.2, macOS Catalina 10.15.6

**Comment by federico-bellucci** [comment]
Same for me
> npm: 6.14.4
> node: 12.16.2
os: Catalina 10.15.6

**Comment by Rudeg** [comment]
Sorry about that, fixed in `0.1.7`
